### PR TITLE
Initial CentOS 8.0 Kickstart

### PR DESCRIPTION
**Pull request by szarkos**

### CODE REVIEW

File: ks/azure/centos80.ks [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+-biosdevname
+-plymouth
+-iprutils
+
+# enable rootfs resize on boot
+cloud-utils-growpart
+gdisk
+
+# add insight-clients
+insights-client
```
Comment (Klaas-) [inline ks/azure/centos80.ks:117]: why do you include that for centos? Isn't that package rhel exclusive (and only makes sense with a rhel subscription)? https://wiki.centos.org/Manuals/ReleaseNotes/CentOS8.1905#head-79555e73a6ccc7e2455a515437302a2084380ba0

Comment (Klaas-) [reply ks/azure/centos80.ks:117]: hmm its in http://olcentgbl.trafficmanager.net/centos/8.0.1905/AppStream/x86_64/os/ :) but still I'd guess without the serverside part it does not make much sense, right?

Comment (szarkos) [reply ks/azure/centos80.ks:117]: Yeah, I can remove that.  It was copy/paste from a RHEL8 kickstart.  Ideally for consistency I'd like to keep the CentOS images somewhat similar to what is being published for RHEL, although we're not bound by that.

Comment (Klaas-) [reply ks/azure/centos80.ks:117]: I agree, I would absolutely follow the rhel8 kickstart - but insights client only makes sense if you push the insights reports to redhat to get analysed there.

**Pull request merged**